src/v3/test/integration/__snapshots__/error-account-creation.test.tsx.snap

```diff
@@ -41,12 +42,12 @@ exports[`error-account-creation should render form 1`] = `
                       <svg
                         aria-hidden="true"
                         class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit emotion-7"
-                        data-testid="InfoOutlinedIcon"
```

Comment:
remove data-testid

Reply:
This is coming from MUI, not any changes on our end. We do not have the ability to apply a test id to SVGs (at least we didnt previously). The infoBox component doesnt have a data-testid defined.